BetterMD/elements/a.py

```
@@ -1,23 +1,52 @@
-from BetterMD.rst.custom_rst import CustomRst
 from .symbol import Symbol
+from ..rst import CustomRst
 from ..markdown import CustomMarkdown
-from ..html import CustomHTML
+import re
 import typing as t
 
-class MD(CustomMarkdown['A']):
+if t.TYPE_CHECKING:
+    from ..parse import Collection
```

Ruff (0.8.2)

`..parse.Collection` imported but unused

Remove unused import: `..parse.Collection`

(F401)